pandas/core/internals/blocks.py

```diff
@@ -2780,44 +2784,39 @@ def is_bool(self):
         return lib.is_bool_array(self.values.ravel())
 
     # TODO: Refactor when convert_objects is removed since there will be 1 path
-    def convert(self, *args, **kwargs):
+    def convert(
```

Comment:
side issue is that a lot of this can be cleaned up now that convert_objects is gone

Reply:
I think that deprecation was before I got here, so it isn't clear to me what else can be cleaned up

Reply:
maybe just remove the comment, you have cleaned up reasonably. you can try looking in history  but if not ok.